### PR TITLE
Allow dash as valid character for regex

### PR DESCRIPTION
**Pull request by crazymind1337**
* adds character - to regex for fact rabbitmq_nodename
* adds unit test for changed regex

### CODE REVIEW

File: spec/unit/facter/util/fact_rabbitmq_nodename_spec.rb [hunk 1]
```diff
@@ -28,6 +28,16 @@
       end
     end
 
+    context 'with dash in username' do
```
Comment (wyardley) [inline spec/unit/facter/util/fact_rabbitmq_nodename_spec.rb]: this should be 'nodename', no?

**Comment by wyardley** [comment]
If I'm understanding correctly, #470 fixed the hostname regex itself, so this is just fixing the nodename fact to match, right?

**Comment by bastelfreak** [comment]
Hi @crazymind1337, thanks for this PR. Can you take a look at the used email address in your commit? It isn't associated with your github account.

File: spec/unit/facter/util/fact_rabbitmq_nodename_spec.rb (hunk 1 above)
Comment (crazymind1337) [reply spec/unit/facter/util/fact_rabbitmq_nodename_spec.rb]: Yes, thanks for the hint. Will be done soon.

**Comment by crazymind1337** [comment]
Oh :) I have worked on that with my company account but created the pull request with my private account :) I gonna fix it soon.

**Comment by crazymind1337** [comment]
@wyardley, there is only one commit. The message just has more than one line. Do you want to have it also just to be a one-liner?

**Comment by wyardley** [comment]
@crazymind1337 Yes, sorry, I just meant to squash if you made a new commit (which looks like you did). Looks good to me, so let's just see if the author looks right to @bastelfreak now.

**Comment by wyardley** [comment]
Merged, thanks for the contribution.

**Pull request merged**